.github/workflows/tests.yml

```diff
@@ -0,0 +1,57 @@
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 3 * * 6'
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8]
+          REQUIREMENTS_LEVEL: [min, pypi]
```

Comment:
Shall we use the same naming as in [`invenio-records`](https://github.com/inveniosoftware/invenio-records/blob/6aba2cb48848f2be8d45a565ad0533095fb091b7/.github/workflows/tests.yml#L24)? Eventually we will push this solution to https://github.com/inveniosoftware/.github/blob/master/workflow-templates/tests.yml.